src/components/trip_view/cal/event_modal/EventModal.js

```diff
@@ -2,24 +2,27 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Modal } from 'react-bootstrap';
 import './EventModal.css';
-import EventInfo from '../../event_view/event_info/EventInfo';
+import EventView from '../../../event_view/EventView';
 
 /**
  * This is a popup modal which displays the event information. It should contain
  * a component that is specific to the type of event being shown.
  */
 export class EventModal extends Component {
   render() {
+    let eventView = this.props.event ? (
+      <EventView event={this.props.event} />
+    ) : (
+      ''
+    );
```

Comment:
New syntax discovery (for reference)! We can also use `false` to mean "don't put anything in this component". So you could do:

```
let eventView = false;
if(this.props.event) eventView = <EventView event={this.props.event} />;
```

(You don't have to do this, it was just a cool discovery!)

Reply:
I like this syntax much more. Long object chains in ternary operators is a headache to read.